main.nf

```diff
@@ -2494,22 +2494,22 @@ process vcf2genome {
 // More complex consensus caller with additional filtering functionality (e.g. for heterozygous calls) to generate SNP tables and other things sometimes used in aDNA bacteria studies
 
 // Create input channel for MultiVCFAnalyzer, possibly mixing with pre-made VCFs.
-if (params.additional_vcf_files == '') {
-    ch_vcfs_for_multivcfanalyzer = ch_ug_for_multivcfanalyzer.map{ it[7] }.collect()
+if (!params.additional_vcf_files) {
+    ch_vcfs_for_multivcfanalyzer = ch_ug_for_multivcfanalyzer.map{ it[-1] }.collect()
```

Comment:
Any reason why you recommend `-1` vs `7`? I went for the latter to enforce very strict structure of the TSV and subsequent tuples?

Reply:
Because I was too lazy to keep track of how many there were :joy: